src/loader-globals-full.ts

```diff
@@ -0,0 +1,15 @@
+// Same as loader-globals.ts except includes rrweb scripts.
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import rrwebRecord from 'rrweb/es/rrweb/packages/rrweb/src/record'
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import 'rrweb/es/rrweb/packages/rrweb/src/plugins/console/record'
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+window.rrweb = { record: rrwebRecord }
```

Comment:
Possibly a nitpick but is it possible to do 
```suggestion
import "./loader-recorder"
```

If not no biggy, but could be a nice way to ensure things stay in sync

Reply:
good call!